core/qthelper.cpp

```diff
@@ -129,6 +129,7 @@ static bool parseCoord(const QString &txt, int &pos, const QString &positives,
 		       const QString &negatives, const QString &others,
 		       double &value)
 {
+	static const QString SEPARATORS = QString(",;");
```

Comment:
I read through CodingStyle.md and used clang-format before I submitted this patch. Normally I would indeed use lowercase/camelcase variables like you suggest, but the variable in this case is a `static const` - vaguely akin to a `#define` in terms of conceptual utility - and I didn't see anything explicit in CodingStyle.md regarding the correct style for static variables. Since in my experience these often have macro-like style (all caps) in C-like codebases, I chose to follow the existing style used for them in the closest adjacent code:

```cpp
bool parseGpsText(const QString &gps_text, double *latitude, double *longitude)
{
        static const QString POS_LAT = QString("+N") + gettextFromC::tr("N");
        static const QString NEG_LAT = QString("-S") + gettextFromC::tr("S");
        static const QString POS_LON = QString("+E") + gettextFromC::tr("E");
        static const QString NEG_LON = QString("-W") + gettextFromC::tr("W");
```
With this in mind - if I've missed the part of CodingStyle.md that clarifies this, can you point it out so I can avoid this mistake in the future? If it's not there, I'm happy to submit a separate patch updating that file.

Also, the word "macro" doesn't appear in CodingStyle.md - would you like me to submit a patch indicating macros should be capitalized as well?;

Reply:
I've changed to lowercase and submitted a separate PR to update CODINGSTYLE.md.